app/models/access.rb

```diff
@@ -53,6 +57,10 @@ def can?(action, model, record = nil)
       end
     end
 
+    def super_admin?
```

Comment:
Moving this up to be a public method so we can delegate from User.
Its useful for doing `user.super_admin?` checks to toggle links etc.